server/www/templates/posts/views.html

```diff
@@ -1,8 +1,17 @@
-<post-view current-view="currentView" filters="filters" base-url="/views/">
-    <h1 translate>post.all_posts</h1>
-
-    <div class="applied-filters">
-        <post-active-filters filters="filters"></post-active-filters>
-        <savedsearch-create filters="filters"></savedsearch-create>
-    </div>
-</post-view>
+<div>
+    <mode-context></mode-context>
+
+    <main role="main">
```

Comment:
Structure ended up a bit ugly here - but needed to get the `<mode-context>` outside the `<main>`